docs/cross-db-migration-design.zh.md

```
@@ -0,0 +1,732 @@
+    local_duckdb:    # ← 逻辑名，用户传给 database= 的值
+      type: duckdb
+      uri: duckdb:///./sample_data/duckdb-demo.duckdb
+    greenplum:       # ← 逻辑名
+      type: greenplum
+      host: 127.0.0.1
+      database: test  # ← 实际库名（由 connector 内部使用）
+```
+
+路由链路：
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)